packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -744,6 +745,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// {@macro flutter.material.Material.clipBehavior}
   final Clip? clipBehavior;
 
+  /// {@template flutter.material.appbar.actionsPadding}
+  /// The padding between the [actions] and the end of the AppBar.
+  ///
+  /// By default, the value of [AppBar.actionsPadding] is [EdgeInsets.zero].
```

Comment:
```suggestion
  /// Defaults to zero.
```

General rule, the documentation should be simple English. As someone who isn't familiar with Flutter and `[EdgeInsets.zero]` might read it first time and might not be able to understand what this means,